app/containers/AddCompanyForm/index.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import * as Yup from 'yup';
+import { createStructuredSelector } from 'reselect';
+
+import injectReducer from 'utils/injectReducer';
+import injectSaga from 'utils/injectSaga';
+import FormComponent from 'components/FormComponent';
+import { selectErrorMessage } from './selectors';
+import { addCompany } from './actions';
+import reducer from './reducer';
+import saga from './saga';
+
+const config = {
+  initialValues: {
+    name: '',
+    address: '',
+    business: '',
+    rating: '',
+    manager: '',
+    active: '',
+  },
+  validationSchema: Yup.object().shape({
+    name: Yup.string('')
+      .email('Invalid email address.')
+      .required('This field is required.'),
+    address: Yup.string().required('This field is required'),
+    business: Yup.string().required('This field is required'),
+    rating: Yup.string(),
+    manager: Yup.string(),
```

Comment:
The user that created the company is the manager, so you can remove this field from the form.